docs/custom/guides/getting-started.md

```diff
@@ -1,6 +1,4 @@
-Title: Getting Started
```

Comment:
This is used by the Wyam docs generator as title metadata, and should not be altered.